### ISSUE TITLE
os.path.rename should be flagged

### ISSUE THREAD

**Issue by sam-s** [PyCQA/pylint#1828]
### Steps to reproduce
Create file `rename-test.py`
```
import os
os.path.rename("a", "b")
```
Run `epylint rename-test.py`

### Current behavior

no errors, no warnings

### Expected behavior

```
error: os.path.rename does not exist
```

### pylint --version output
```
pylint 1.8.1, 
astroid 1.6.0
Python 2.7.14 (default, Jan  6 2018, 12:16:16) 
[GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.39.2)]
```

**Comment by ddaanet** [comment]
Indeed, it appears that (with the default configuration) E1101 (no-member) is disabled on os.path.

```
import os.path

os.path.foo()  # no problem!
os.foo()  # no-member
```

**Comment by PCManticore** [comment]
Thanks for the issue. This just happens on Python 2, because we infer `os.path.rename` as being what we call an Uninferable object. that is, the inference engine thinks it is defined there, although it is not. This does not happen in Python 3, and I imagine because of some difference between how `os.path` is arranged vs in Python 2.
Nevertheless, you can still trigger this error if you want, just that it is disabled by default, by running: `python2 -m pylint a.py --ignore-on-opaque-inference=n`
